Added support for Mobile data sources/components

rubinatorz opened the pull request:
Hey @Cyb3rWard0g !

ATT&CK v13 is released this week, including Mobile data sources. I've added the mobile data sources and components to attackcti. Hope you have time to merge this soon 🤓

Cyb3rWard0g commented:
Thank you very much @rubinatorz ! Sorry for the late response. Taking a look at it today :) I appreciate your contributions! I will update the library soon.

rubinatorz commented:
Thanks @Cyb3rWard0g

Today I've added a few more things: using enrich_techniques_data_sources within the get_mobile_techniques and get_ics_techniques (get_enterprise_techniques was already using this). 

Within enrich_techniques_data_sources I removed the check on x_mitre_data_sources. This attribute is not used/updated anymore by MITRE. It not there for the new Mobile data sources for example. See https://github.com/mitre/cti/issues/213.

For the enrichment in enrich_techniques_data_sources it still uses the x_mitre_data_sources attribute, also for Mobile to keep it like it is now. But maybe for the future, we need to think on what to do with x_mitre_data_sources as MITRE discontinued the use of this.

Cyb3rWard0g reviewed (approved):
Thank you very much @rubinatorz ! Looks great! sorry for the delay. I got sidetracked with something at work and a conference.
